### DISCUSSION TITLE
Spoken Languages Feature

### DISCUSSION THREAD

**Discussion by verilyanton** [plante-app-team/plante#283] [Features]
Currently, there is a limitation on the languages of the products users can add. If the package is not in one of the languages they have marked as spoken by them, they cannot add it.

From one point of view, it is reasonable—if you don't know the language of the product, you probably can't understand whether it's vegan or not.

However, there are cases when you can, for example, if there is a vegan label from a reputable vegan certifier (like EVU's V-Label), so this becomes a false negative. Moreover, this can enhance the experience of finding vegan food abroad.

**Comment by blazern** [comment]
I agree, that might come handy in some situation.
Maybe the person even wants to add the product to the app so that moderators would verify its status and the person would be able to buy it a bit later.

The feature needs to be thought throughout though:
- Do we just allow to add the product in any language?
- Or do we show a huge warning and asking the person if they're sure they want to do that?
- Do we display all the products fields, or do we not let some fields to be filled?
- Or do we add some additional fields?

It would be great if someone came with suggestions on how exactly this should be implemented.

**Reply by verilyanton** [reply]
This is certainly not a top priority, and given that the new team members are not familiar with this kind of implementation details yet, let's take time to investigate it

**Reply by SergioB** [reply]
What is our ideal use case? 
We want it to be easy for users (cause we need more contributing users) so it's better to make it as simple as possible and let the moderators later edit or reject it.
